environment/s3.tf

```diff
@@ -31,6 +31,14 @@ resource "aws_s3_bucket" "pa_uploads" {
     }
   }
 
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = "aws:kms"
```

Comment:
Don't you want this to be AES256 to match the bucket policy, same for the other one?

Reply:
so under the hood aws kms seems to use aes 256 and i thought it needed a bespoke key but yeah you're right. seems it doesn't any more and actually that seems to pass all tests fine so yes that's a very good point.